.gitignore

```diff
@@ -1,19 +1,13 @@
-/.phpunit.cache
 /node_modules
-/public/build
-/public/hot
 /public/storage
 /storage/*.key
 /vendor
 .env
 .env.backup
-.env.production
 .phpunit.result.cache
-Homestead.json
-Homestead.yaml
-auth.json
 npm-debug.log
 yarn-error.log
-/.fleet
 /.idea
 /.vscode
+/package-lock.json
+/composer.lock
```

Comment:
I understand that when we run composer install, it re-generates lock files so I have excluded from git. keeping in mind we should not use composer update. the reason behind including this is, if any current version has minor issue then it will fetch latest one.

Reply:
The purpose of lock files is so that everyone on the team is installing the same versions, since the package/composer files have non-specific version constraints. Same goes for CI/CD. So they very much should be included.